services-directions/src/main/java/com/mapbox/api/directions/v5/models/BannerComponents.java

```diff
@@ -107,6 +107,18 @@ public static Builder builder() {
   @SerializedName("imageBaseURL")
   public abstract String imageBaseUrl();
 
+  /**
+   * An image associated with the banner component. Unlike the imageBaseUrl,
+   * this image url does not include image density encodings.
```

Comment:
Mention of guidance/junction view here. Developers might wonder what image could be associated with the banner component.

Reply:
yeah first attempt was to make it generic but, it's not generic yet so comment would be better specific.

here's another iteration. also feel free to post a suggestion :)
```
   * In some cases when the {@link StepManeuver} will be difficult to navigate, an image
   * can describe how to proceed. The domain name for this image is a Junction View.
   * Unlike the imageBaseUrl, this image url does not include image density encodings.
```

Reply:
👍